game/src/backend/gamestate.rs

```diff
@@ -184,6 +185,20 @@ impl GameState {
             name
         )))
     }
+    /// Deletes all files in the directory saves, returns Ok if saves directory does not exist
+    pub(crate) fn delete_saves() -> RLResult {
+        let existing_files = fs::read_dir("./saves");
+        if existing_files.is_err() {
+            return Ok(());
+        }
+        for entry in existing_files.unwrap() {
+            let file = entry.unwrap();
+            if file.metadata().unwrap().is_file() {
+                fs::remove_file(file.path()).expect("one savefile to be deleted");
+            }
+        }
+        return Ok(());
```

Comment:
wir können in dieser Zeile nur Ok(()) schreiben, weil wir in Rust das return nicht brauchen, wenn wir das Semicolon weglassen